steward/src/somm_send.rs

```diff
@@ -1,9 +1,9 @@
 use deep_space::coin::Coin;
 use deep_space::error::CosmosGrpcError;
-use deep_space::private_key::PrivateKey as CosmosPrivateKey;
 use deep_space::Contact;
 use deep_space::Fee;
 use deep_space::Msg;
+use gravity_bridge::cosmos_gravity::crypto::PrivateKey as CosmosPrivateKey;
```

Comment:
Same thing here, let's not use this type unless a gravity bridge function specifically requires it.